python/tritonserver/_api/_server.py

```diff
@@ -621,6 +621,9 @@ def stop(self) -> None:
         self._server.stop()
         self._server = Server._UnstartedServer()
 
+    def get_c_ptr(self):
```

Comment:
question: 

1) can we make this a private method: 
```_ptr(self):```

2) Is there a way to pass the pybind object as an opaque pointer to the other components instead of accessing the pointer from python?